packages/lit/README.md

```diff
@@ -1,14 +1,18 @@
-# Lit 2.0
+# Lit
 
-Fast, lightweight web components
+## Simple. Fast. Web Components.
+
+Lit is a simple library for building fast, lightweight web components.
+
+At its core is a boilerplate-killing component base class that provides reactive state, scoped styles, and a declarative template system that leads the pack in size, speed, and expressiveness.
```

Comment:
"At its core *Lit* is"

Reply:
I agree this phrasing is a bit of a mouthful, but I think it's correct, at least as intended.

Lit is a library. At its core is a base class.

(like: Earth is a planet. At its core is molten lava.)

Reply:
Oh I see! I didn't parse it like that. How about "At Lit's core"?